MekHQ/src/mekhq/resources/CampaignOptionsDialog.properties

```diff
@@ -307,3 +308,5 @@ chkCapturePrisoners.text=Capture Prisoners in Scenarios
 chkCapturePrisoners.toolTipText=If checked, you will have the opportunity to capture prisoners when resolving scenarios
 chkAssignPortraitOnRoleChange.text=Automatically assign portrait on role change
 chkAssignPortraitOnRoleChange.toolTipText=With this enabled, a person without a portrait will automatically gain a random portrait when their primary role is switched to one of those selected below
+chkShowHistoricalDailyReport.text=Show the historical daily report for each gaming session
```

Comment:
Seems a little verbose. Maybe "Temporarily retain daily log history", or something like that?